### ISSUE TITLE
Cloning doesn't seem to pull down images or stylesheets locally

### ISSUE THREAD

**Issue by ian** [brovikov/scraper-class#1]
All I see is ebay.html, no images or stylesheets

**Comment by brovikov** [comment]
@ian , I updated ruby class. But for now it looks like not very simple task :) for 2 hours. Because, even Google Chrome some time make "Save is.." task incorrect.  

So it would be great if you tell me our final goal :)

**Comment by brovikov** [comment]
@ian , any comments?

**Comment by ian** [comment]
Did you update this since I last looked at it?

**Comment by brovikov** [comment]
Yep, it's updated. Now it store images/css/js to separate directories.

**Comment by ian** [comment]
Questions:

1) I see you maek img, js and css directories.  Do you rewrite the HTML for
these saved assets?
2) Do you have any thoughts on rewriting the urls for the anchor tags?

**Comment by brovikov** [comment]
Yep, it rewrites HTML, so it's allow to open page locally with CSS/JS/IMG applied.
 Anchors also should be work.

But it still not working good for JS heavy sites, SItes which detect bots and makes redirects, and so on..
If we writing an universal tool - it's a complex not trivial task.

**Comment by ian** [comment]
Is there any way to remove all the analytics js? (GA, mixpanel, etc)

**Comment by brovikov** [comment]
Yep, we can do it (but we need strong requirements what exactly we need remove. "etc" doesn't work.

**Comment by brovikov** [comment]
May be is better extract and save to DB some data? Or make a img snapshot for the page? I can't understand why you need store the whole page?

**Comment by brovikov** [comment]
So, what we will do? What is next step?

**Comment by ian** [comment]
I will invite you to the main app now but you will need to figure out how
to set it up.  It's complicated, and I haven't written documentation for
installing it yet.
